app/views/pages/home.html.erb

```diff
@@ -1 +1,55 @@
-<h1>Welcome to Quizzland!</h1>
+<header>
+  <%= render "./layouts/partials/navbar" %>
+
+  <div class="relative overflow-hidden bg-no-repeat bg-cover" style="
+    background-position: 50%;
+    background-image: url('https://www.nexus-analytics.com.my/wp-content/uploads/2020/10/knowledge-banner.jpg');
+    height: 350px;">
+    <div class="absolute top-0 right-0 bottom-0 left-0 w-full h-full overflow-hidden bg-fixed"
+      style="background-color: rgba(0, 0, 0, 0.35)">
+      <div class="flex justify-center items-center h-full">
+        <div class="text-center text-white px-6 md:px-12">
+          <h1 class="text-5xl font-bold mt-0 mb-6"><%= t("welcome") %></h1>
+          <h3 class="text-3xl font-bold mb-8"><%= t("to_quizland") %></h3>
+          <%= link_to t("first_quiz"), new_quiz_path, class:"inline-block px-6 py-2.5 border-2 border-white text-white font-medium text-xs leading-tight uppercase rounded hover:bg-black hover:bg-opacity-5 focus:outline-none focus:ring-0 transition duration-150 ease-in-out" %>
+        </div>
+      </div>
+    </div>
+  </div>
+</header>
+
+<h2 class="font-bold text-xl m-8"><%= t('why_quizland') %></h2>
+
+<div class="p-10 grid grid-cols-1 sm:grid-cols-1 md:grid-cols-3 lg:grid-cols-3 xl:grid-cols-3 gap-5">
+    <div class="rounded overflow-hidden shadow-lg">
+      <div class="px-6 py-4">
+        <div class="font-bold text-xl mb-2"><%= t("fast") %></div>
+        <p class="text-gray-700 text-base">
+          <%= t("fast_text") %>
+        </p>
+      </div>
+    </div>
+    <div class="rounded overflow-hidden shadow-lg">
+      <div class="px-6 py-4">
+        <div class="font-bold text-xl mb-2"><%= t("comfortably") %></div>
+        <p class="text-gray-700 text-base">
+          <%= t("comfortably_text") %>
+        </p>
+      </div>
+    </div>
+
+    <div class="rounded overflow-hidden shadow-lg">
+      <div class="px-6 py-4">
+        <div class="font-bold text-xl mb-2"><%= t("profitably") %></div>
+        <p class="text-gray-700 text-base">
+          <%= t("profitably_text") %>
+        </p>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="flex justify-center items-center h-full">
+  <%= link_to t("quizzes.all_quizzes"), quizzes_path, class:"inline-block px-6 py-2.5 border-2 border-black text-black font-medium text-xs leading-tight uppercase rounded hover:bg-black hover:bg-opacity-5 focus:outline-none focus:ring-0 transition duration-150 ease-in-out m-5" %>
+</div>
+
+<%= render "./layouts/partials/footer" %>
```

Comment:
do you need footer only at home page?
if everywhere, then render it in `application.html.erb`